hackmcgill/dashboard
# Added country of residence field to Personal Details section of hacker application

### Tickets:
https://github.com/hackmcgill/dashboard/issues/1009

### List of changes:
I added country to all the files, see details in the issue linked above. Validation still needs to be added to hackerAPI.

### Type of change:
- [ X] New feature (non-breaking change which adds functionality)

### How to test:
Fill out a hacker application and check that the country field is in Personal Details and works properly. Check that it went through to mongoDB.
